docs/sources/panels/visualizations/table/_index.md

```diff
@@ -0,0 +1,34 @@
++++
+title = "Table panel"
+keywords = ["grafana", "dashboard", "documentation", "panels", "table panel"]
+type = "docs"
+aliases = ["/docs/grafana/latest/reference/table/", "/docs/grafana/latest/features/panels/table_panel/"]
+[menu.docs]
+name = "Table panel"
+parent = "visualizations"
+weight = 1000
++++
+
+# Table panel
+
+The table panel is very flexible, supporting multiple modes for time series and for tables, annotation, and raw JSON data. This panel also provides date formatting, value formatting, and coloring options.
+
+{{< docs-imagebox img="/img/docs/tables/table_visualization.png" max-width="1200px" lightbox="true" caption="Table visualization" >}}
+
+> **Note:** Annotations are not currently supported in the table panel.
+
+## Data and field options
+
+Table visualizations allow you to apply:
+
+- [Data transformations]({{< relref "../../transformations.md" >}})
+- [Field options and overrides]({{< relref "../../field-options/_index.md" >}})
+- [Thresholds]({{< relref "../../thresholds.md" >}})
+
+## Display options
+
+- **Show header -** Show or hide column names imported from your data source.
+- **Sort ascending/descending -** Click a column title to change the sort order from default to descending to ascending. Each time you click, the sort order changes to the next option in the cycle. You can only sort by one column at a time.
+  {{< docs-imagebox img="/img/docs/tables/sort-descending.png" max-width="500px" caption="Sort descending" class="docs-image--no-shadow" >}}
+- [Table field options]({{< relref "table-field-options.md" >}}) allow you to change [field options]({{< relref "../../field-options/_index.md" >}}) such as column width, alignment, and cell display mode.
+- [Filter table columns]({{< relref "filter-table-columns.md" >}}) - You can temporarily change how column data is displayed. For example, you can order values from highest to lowest or hide specific values.
```

Comment:
I would like to still make these options just part of this page, they are all part of the same task (configuring table display options). Not having them on the table documentation page feels like a missed opportunity.

Reply:
Two documentarians out of two surveyed disagree. They are linked from the page, but they should be separate topics.

Reply:
Between the two of us, @achatterjee-grafana and I have close to three decades of experience with technical documentation. There is a method to what we do and how we do it.

Reply:
Is there some common ground or compromise that we could discuss as a way forward here?

Reply:
I respect your experience in technical documentation, I really do. But I think some domain knowledge is also required in order to logically group information. 

The current table documentation page: https://grafana.com/docs/grafana/latest/panels/visualizations/table-panel/ 

is far from a long page and in need of splitting up. The main purpose of this page is to list and document the different display options for the table panel. Not including the main display options (display mode) and hiding this on a sub-page is not very user friendly. Scrolling a tiny bit is much easier than finding a cryptic link "Table field options". Each page has a TOC on the side as well for easy navigation within larger documents. Each section heading will be indexed so they can be directly discovered via search. I see mostly downsides by splitting this page up. 

Many visualizations will have field display options, some will only have field display options. Splitting display options up in two pages feels ad-hoc and confusing. I know these options are split into two tabs in the UI, but that is to me an argument to document them together so it's clear that there are display options in the field/override tabs as well.

Reply:
Any pattern we apply should be a pattern we can apply to all the visualizations. True, table is short. Many of the other visualizations are not.

A longer term goal is to expand all of the "main" visualization topics with more concept information. Things like why you would want to use this visualization, which data is best suited for it, examples, and so on. Also, the current field description is sadly lacking.

My goal when I rewrote the Panels section for 7.0 was:
- Each visualization has a page.
- Each field is defined.

That was a pretty low bar, but it was one that we were not meeting at that time. There is a lot more work to do, on all of them.

Your suggestion of putting all the display options into the main Table section might work in the short term, even if it would be contrary to what we know to be good documentation practices. However, it would require more rework in the future when we add the "deep dives" and expanded guidance that this section needs.

Regarding the purpose of the page, the panel display fields would ideally be in a different page and have expanded descriptions and explanations for all the visualizations, including table. We don't have the bandwidth for that right now, but it is on the To Do list.

Reply:
Ok, fine. Just wished to have the display options listed together. If we can mention cell display mode and show examples on the main page later that would be great. I would like the main visualization page to be a source of inspiration and show different settings & examples. So documenting the settings on another page could be a good idea. Just splitting up display options and field options feel bad, they are both display options just different levels. 

Tested this PR but the side menu navigation did not work, all the links are wrong 
![Screenshot from 2020-09-23 17-47-24](https://user-images.githubusercontent.com/10999/94036868-0c999d80-fdc5-11ea-97cf-5566d67c432a.png)

Reply:
ARGH, thought I had fixed all the merge issues. Will check the side panel momentarily.

The longer term plan is to have the main page for each visualization have what you are talking about:
- What the visualization is
- How to use it, what data is best suited to it
- Use cases and examples of each visualization used well
- Links to tasks (such as Add and set up a table, which we currently don't have)
- Links to reference documentation. The list of fields and descriptions that are currently in the visualization topics is woefully incomplete. Most of them need to be expanded and possibly illustrated with images, explanations, and use cases. 

So...yeah. We have plans. Just trying to get pieces into place as we touch them right now, laying the groundwork for the future.

Reply:
Heh. Whoops, that menu thing wasn't a merge problem, was a me problem. It looked right when I tested it before...oh, well. Fixing now.